### ISSUE TITLE
'N' and 'SHIFT+N' keyboard shortcuts to cycle through the digitizing common angle options do not work

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS#53713]
### What is the bug or the crash?

#52988 added `N` and `SHIFT+N` keyboard shortcuts to cycle through the common angle options of the advanced digitizing panel "Snap to common angles" menu.
This doesn't seem to work.

### Steps to reproduce the issue

Enable advanced digitizing panel
Click and expand the "Snap to common angles" menu on the toolbar
Select a list of angle
Try to cycle through pressing `N` or `shift+N`

### Versions

Tested on Windows 10 version 2009, QGIS 3.33 7df22c4d1e

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by elpaso** [comment]
Works for me on Linux, are you sure you have the floater open and you are actually in the process of digitizing something? I will be able to test on windows in a couple of weeks.

**Comment by DelazJ** [comment]
Aaaah... not at all. I was not digitizing. I thought the shortcut would select another entry in the drop-down menu (anytime), not that it does that during edit operation. I could retry in a couple of days when I'm on Windows.

**Comment by DelazJ** [comment]
Well I gave it a shot, and I wonder if the tooltip is correct. It states:
> press n to cycle through the options

To me, that means, if I press N, I will move to the next common angles item in the list (ie `1.0,...` --> `5.0,...`--> `10.0,...`). But in my experience, it cycles through my **history** of the common angles usage. So I can get `1.0,...` --> `30.0,...`--> `45.0,...` depending on what I used. Am I correct?

**Comment by elpaso** [comment]
No, the N / SHIFT+N cycle through the action items in the list so it should not jump around but go sequentially, I'll check it put on windows shortly.
